soc/arm/silabs_exx32/Kconfig

```diff
@@ -26,6 +26,11 @@ config SOC_PART_NUMBER
 	  that you should not set directly. The part number selection choice defines
 	  the default value for this string.
 
+config CONFIG_HAS_GECKO_DCDC
```

Comment:
should be 'config HAS_GECKO_DCDC'.  The CONFIG_... gets added by build system and is what's used in the source code.